wom/src/main/scala/wom/package.scala

```diff
@@ -1,7 +1,44 @@
+package wom
 
-import wdl.values.WdlValue
+import org.apache.commons.codec.digest.DigestUtils
 import wom.callable.Callable.InputDefinition
+import wom.values.WdlValue
+
+import scala.util.Try
+
+trait TsvSerializable {
+  def tsvSerialize: Try[String]
+}
+
+class WdlExpressionException(message: String = null, cause: Throwable = null) extends RuntimeException(message, cause)
```

Comment:
WdlExpressionException doesn't belong in the `wom` package object

Reply:
It's `WomExpressionException` in part II if that helps... 😛